src/main/frontend/extensions/slide.cljs

```diff
@@ -63,9 +64,9 @@
   [loading? style config blocks]
   [:div
    [:p.text-sm
-    [:span.opacity-70 "Tip: press "]
+    [:span.opacity-70 (t :page/slide-view-tips)]
     [:code "f"]
-    [:span.opacity-70 " to go fullscreen"]]
+    [:span.opacity-70 (t :page/slide-view-go-fullscreen)]]
```

Comment:
@logseq-cldwalker, These separate strings are everywhere. How can it be improved? Like this?: ad26767f0e226eef64e82ed9a108ab4a51f38902

Reply:
@queeup Do you know how this ":code" work? I don't know so I cannot modify it...

Reply:
Not sure. Thats why I mention to cldwalker. He will show you how. I hope.

Reply:
Look at here please: https://github.com/logseq/logseq/pull/9714/commits/a1a18a1b0816af9fe2e20c8ef989329097449f3f